audio-worklet/basic/audio-worklet-node-options/index.html

```diff
@@ -0,0 +1,106 @@
+<!--
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <meta name="mobile-web-app-capable" content="yes">
+  <link href="../../../assets/was-styles.css" rel="stylesheet">
+  <title>AudioWorkletNode Options! | WebAudio Samples</title>
+</head>
+
+<body>
+  <div class="container was-page-wrap"></div>
+  <script type="module">
+    import PageData from './PageData.js';
+
+    import {
+      html, render,
+      Component, WorkletIndicator, TopBar, Footer,
+      OneColumnView
+    }
+      from '../../../assets/Components.js';
+
+
+    render(html`
+      <div id="TopBar"></div>
+      <div id="WorkletIndicator"></div>
+      <div id="Description"></div>
+      <div id="DemoInput">
+        <label for="type">Type:</label>
+        <select name="type" id="demo-input-type">
+          <option value="noise">Noise</option>
+          <option value="sine">Sine</option>
+          <option value="square">Square</option>
+          <option value="triangle">Triangle</option>
+          <option value="sawtooth">Sawtooth</option>
+        </select>
+        <label for="frequency">Frequency (1 - 20000):</label>
+        <input type="number" name="frequency" id="demo-input-frequency" min="1" max="20000" value=440>
```

Comment:
Didn't count, but we wrap the code at column 80:
https://google.github.io/styleguide/jsguide.html#formatting-column-limit